### DISCUSSION TITLE
Refactor to make it easier to work on Claude Dev with Claude Dev

### DISCUSSION THREAD

**Discussion by quinn** [cline/cline#327] [Feature Requests]
Currently I'm hitting my token rate limit almost immediately after the first read_file execution on src/ClaudeDev. I'd be happy to work on refactoring src/ClaudeDev.ts into smaller files if that seems helpful.

**Comment by philikai** [comment]
+1
I was able to add a feature to Cline, but damn, those files are long and you get lost super easily. Refactoring is needed.
